src/components/DocTabsRadio/DocTabsRadioSection.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Radio,
+  RadioGroup
+} from '@material-ui/core';
+import React, { memo, useContext } from 'react';
+import { SheetCtx } from '../../contexts/sheet.context';
+import { Sheet, Spreadsheet } from '../../models';
+import { getSheetByTitle } from '../../utils/spreadsheet';
+
+interface OwnProps {
+  readonly spreadsheet: Spreadsheet;
+};
+type Props = OwnProps;
+
+const DocTabsRadioSection = ({spreadsheet} : Props) => {
+  const {sheet, setSheet} = useContext(SheetCtx);
+  const initSheet = sheet ? sheet : spreadsheet.sheets[0];
+  setSheet(initSheet);
+
+  const assignNewSheet = (sheetToAssign: Sheet) => {
+    const newSheet = Object.assign({}, sheet);
```

Comment:
:bug: Use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) 

```suggestion
    const newSheet = {...sheet,
        title: sheetToAssign.title,
        usersData: sheetToAssign.usersData,
        variables: sheetToAssign.variables
    };
```